tests/WebServerE2ETests/nanoFramework WebServer E2E Tests.postman_collection.json

```diff
@@ -0,0 +1,888 @@
+		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json",
+		"_exporter_id": "13804166"
+	},
+	"item": [
+		{
+			"name": "ServerCommandReceived_EmptyPath",
+			"event": [
+				{
+					"listen": "test",
+					"script": {
+						"exec": [
+							"pm.test(\"Status 200\", function () {",
+							"    pm.response.to.have.status(200);",
+							"});",
+							"",
+							"pm.test(\"Response text\", function () {",
+							"    pm.expect(pm.response.text()).to.include(\"Hi from nanoFramework Server\");",
+							"});"
+						],
+						"type": "text/javascript"
+					}
+				}
+			],
+			"request": {
+				"method": "GET",
+				"header": [],
+				"url": {
+					"raw": "{{base_url}}",
+					"host": [
+						"{{base_url}}"
+					]
+				},
+				"description": "This is a GET request and it is used to \"get\" data from an endpoint. There is no request body for a GET request, but you can use query parameters to help specify the resource you want data on (e.g., in this request, we have `id=1`).\n\nA successful GET response will have a `200 OK` status, and should include some kind of response body - for example, HTML web content or JSON data."
+			},
+			"response": []
+		},
+		{
+			"name": "ServerCommandReceived_Params",
+			"event": [
+				{
+					"listen": "test",
+					"script": {
+						"exec": [
+							"pm.test(\"Status OK\", function () {\r",
+							"    pm.response.to.have.status(200);\r",
+							"});\r",
+							"\r",
+							"pm.test(\"Body contains parameters\", function () {\r",
+							"     pm.expect(pm.response.text()).to.include(\"Here are the parameters of this URL: <br />Parameter name: testKey1, Value: 123<br />Parameter name: testKey2, Value: 321<br />\")\r",
+							"});"
+						],
+						"type": "text/javascript"
+					}
+				}
+			],
+			"request": {
+				"method": "GET",
+				"header": [],
+				"url": {
+					"raw": "{{base_url}}/param.htm?testKey1=123&testKey2=321",
+					"host": [
+						"{{base_url}}"
+					],
+					"path": [
+						"param.htm"
+					],
+					"query": [
+						{
+							"key": "testKey1",
+							"value": "123"
+						},
+						{
+							"key": "testKey2",
+							"value": "321"
+						}
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "ServerCommandReceived_WriteFile",
+			"event": [
+				{
+					"listen": "test",
+					"script": {
+						"exec": [
+							"pm.test(\"Status OK\", function () {\r",
+							"    pm.response.to.have.status(200);\r",
+							"});"
+						],
+						"type": "text/javascript"
+					}
+				}
+			],
+			"request": {
+				"method": "GET",
+				"header": [],
+				"url": {
+					"raw": "{{base_url}}/useinternal",
+					"host": [
+						"{{base_url}}"
+					],
+					"path": [
+						"useinternal"
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "[NotWorking] ServerCommandReceived_ReadFile",
```

Comment:
I've been quickly looking at this and I've been added one call to create the file first. The file exist and when using a browser:
![image](https://github.com/nanoframework/nanoFramework.WebServer/assets/8145578/624e9e39-0d54-4713-a39b-4418c339a6a9)
So not sure exactly why I get this
![image](https://github.com/nanoframework/nanoFramework.WebServer/assets/8145578/3be72175-88ee-4bb5-9f38-120c4e83842e)
:

Reply:
I fixed it, I'll raise a PR

Reply:
Yep, that was the case - file in browser works in postman there was aborted. But you fixed it :)